src/spec-common/injectHeadless.ts

```diff
@@ -319,7 +320,9 @@ export async function runPostCreateCommands(params: ResolverParameters, containe
 		return 'skipNonBlocking';
 	}
 
-	await runPostAttachCommand(params, containerProperties, config, remoteEnv);
+	if (!params.skipPostAttach) {
+		await runPostAttachCommand(params, containerProperties, config, remoteEnv);
```

Comment:
Should we return a different status from this method if we skipped post attach? It's not clear to me how the return value is used.

Reply:
We could return 'skipPostAttach' and handle it [here](https://github.com/microsoft/vscode-remote-containers/blob/19a2dc727aa2d1feb3983ffff345ed7341dc2950/src/common/injectHeadless.ts#L596). It wouldn't change what we do there (except for accepting the new value) though and we can add the additional return value later if we need.

Reply:
Sounds good on waiting until we need it 👍